### ISSUE TITLE
Work With Other Datasets

### ISSUE THREAD

**Issue by antmarakis** [allenai/cartography#4]
Hi! This looks like a very interesting tool, I am wondering if it would be easy to use on other datasets. I see only GLUE/NLI datasets are supported. Do you have any tips on how to use this on a simple {TEXT, LABEL} task? Thanks!

**Comment by douglashiwo** [comment]
I have the same question with antmarakis. Can you kindly help?

**Comment by lukasmoldon** [comment]
Just sharing my experience with this repo, maybe this helps someone in the future:

I used this repo for a {TEXT, LABEL} task with BERT models. Since neither this type of task nor this type of model is supported in the training section of this repo, I would recommend to first train any model on any dataset on your own (without using the code of this repo). While training, save somewhere the logits of each data instance together with the gold standard label and a unique identifier, [as suggested by the authors (see "_Note:_" section)](https://github.com/allenai/cartography#train-glue-style-model-and-compute-training-dynamics).

After training you can use [train_dy_filtering](https://github.com/allenai/cartography/blob/main/cartography/selection/train_dy_filtering.py), as explained [here](https://github.com/allenai/cartography#plot-data-maps) to generate DataMaps and to obtain coordinates for further data filtering. You just need to extend [this line of code](https://github.com/allenai/cartography/blob/main/cartography/selection/train_dy_filtering.py#L362) by any additional name, which you use from now on as task name. Then you can call `python -m cartography.selection.train_dy_filtering --plot --task_name "YOUR_NEW_TASK_NAME" --model "ANY_NAME_YOU_WANT" --model_dir ""` from the main directory of this repo to create the DataMap. Make sure to create a `training_dynamics` folder in the main directory including your training dynamics. The plot will be automatically saved in the cartography folder, while the coordinates will be stored to the main directory (this can be changed by other arguments like --plots_dir).

**Comment by pskadasi** [comment]
fixing errors of this repo is more time consuming than extracting training dynamics from the model that is trained independently :)
